packages/block-library/src/media-text/save.js

```diff
@@ -33,9 +33,29 @@ export default function save( { attributes } ) {
 		verticalAlignment,
 		imageFill,
 		focalPoint,
+		linkClass,
+		href,
+		linkTarget,
+		rel,
 	} = attributes;
+	const newRel = isEmpty( rel ) ? undefined : rel;
+	const image = <img src={ mediaUrl } alt={ mediaAlt } className={ ( mediaId && mediaType === 'image' ) ? `wp-image-${ mediaId }` : null } />;
+	const imageMedia = (
+		<>
+			{ href ? (
+				<a
+					className={ linkClass }
+					href={ href }
+					target={ linkTarget }
+					rel={ newRel }
+				>
+					{ image }
+				</a>
+			) : image }
+		</>
+	);
```

Comment:
Thinking this might be a little easier to read if we remove the ternary statement and add some indentation:

```js
let image = (
	<img
		src={ mediaUrl }
		alt={ mediaAlt }
		className={ ( mediaId && mediaType === 'image' ) ? `wp-image-${ mediaId }` : undefined }
	/>
);

if ( href ) {
	image = (
		<a
 			className={ linkClass }
 			href={ href }
 			target={ linkTarget }
 			rel={ newRel }
 		>
			{ image }
		</a>
	);
}
```

Reply:
Yeah, because they are not available yet :) Would be great to add them on transform now as well. Thanks!